packages/core/src/GreenButton.js

```diff
@@ -1,19 +1,7 @@
-import styled from 'styled-components'
+import React from 'react'
 import Button from './Button'
-import theme from './theme'
 
-const GreenButton = styled(Button)`
-  background-color: ${props => props.theme.colors.green};
-
-  &:hover {
-    background-color: ${props =>
-      props.disabled ? null : props.theme.colors.darkGreen};
-  }
-`
-
-GreenButton.defaultProps = {
-  theme: theme
-}
+const GreenButton = props => <Button variant="secondary" {...props} />
```

Comment:
GreenButton should be deprecated in favor of Button with a variant prop. Add a deprecation warning